{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "586a8584_a5f66286",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1297197
      },
      "writtenOn": "2021-06-03T20:40:40Z",
      "side": 1,
      "message": "Are these only timing out locally, or are they flaky on CQ also?",
      "revId": "bf2777b6457fa4aba7696ea0f62feb05283f4453",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "d2f786cd_2add26c4",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1256237
      },
      "writtenOn": "2021-06-03T20:47:11Z",
      "side": 1,
      "message": "It\u0027s difficult to tell.  You have to reduce the batch size to 1 to isolate which test is failing.  The bots already have a bunch of other errors.  I got failures locally, removed a test, then the script would pass once and fail on next run. So turned off one of the subtests, then another, then just turned off the block.",
      "parentUuid": "586a8584_a5f66286",
      "revId": "bf2777b6457fa4aba7696ea0f62feb05283f4453",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}